src/main.cc

```diff
@@ -135,6 +140,7 @@ void print_usage(FILE* out) {
       "  -T, --dump-at=TIME         dump memory at global timepoint TIME\n"
       "  -V, --verbose              log messages that may not be urgently \n"
       "                             critical to the user\n"
+      "  -v, --version              print the version number and exit\n"
```

Comment:
I'm trying to keep all global single-letter options as capitals. Here I think we should just not have a single-letter option.